README.md

```diff
@@ -368,6 +368,8 @@ The following table lists the configurable parameters for this chart and their d
 | `extraInitContainers`                           | Additional init containers to run before starting main containers   | `[]`                                         |
 | `worker`                                        | Worker specific variables. Most global variables also apply here.   | *see `values.yaml`*                          |
 | `housekeeping.enabled`                          | Whether the [Housekeeping][housekeeping] `CronJob` should be active | `true`                                       |
+| `housekeeping.args`.                            | Custom start up arguments for netbox container arguments.           | `[]`                                         |
+| `housekeeping.command`                          | Custom container entrypoint.                                        | ["/opt/netbox/venv/bin/python","/opt/netbox/netbox/manage.py",rqworker] |
```

Comment:
Nitpick: can you please swap these two lines? Logically I feel like that makes more sense as the `args` are appended to the `command` when both are present.